spec/paperclip/has_attached_file_spec.rb

```diff
@@ -38,6 +38,15 @@
       assert_adding_attachment('avatar').defines_callback('after_commit')
     end
 
+    context 'when the class does not allow after_commit callbacks' do
+      it 'defines an after_destroy callback' do
+        assert_adding_attachment(
+          'avatar',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.